lldb/source/Utility/XcodeSDK.cpp

```diff
@@ -259,6 +259,27 @@ bool XcodeSDK::SupportsSwift() const {
   }
 }
 
+bool XcodeSDK::SDKSupportsBuiltinModules(const llvm::Triple &target_triple,
```

Comment:
This function got changed in https://github.com/llvm/llvm-project/pull/102239

Reply:
I already accounted for the version check changes

Reply:
But you didn't pick up the environment check

Reply:
Don't think we really want to support this for non-native environments tbh